.coveragerc

```diff
@@ -404,6 +404,7 @@ omit =
     homeassistant/components/feedreader.py
     homeassistant/components/foursquare.py
     homeassistant/components/goalfeed.py
+    homeassistant/components/homekit_controller.py
```

Comment:
Since we have light and switch platforms now, let's move it to the top of the `coveragerc` file and make the platform matcher use a wildcard (`*`)

Reply:
Done